web/client/components/mapcontrols/annotations/AnnotationsEditor.jsx

```diff
@@ -102,8 +102,13 @@ const bbox = require('@turf/bbox');
  * @prop {string} pointType the type of the point, values are "marker" or "symbol"
  * @prop {object[]} lineDashOptions list of options for dashed lines
  * @prop {object[]} symbolList list of symbols
- * @prop {string} defaultShape default shape for symbol
  * @prop {string} symbolsPath path to the svg folder
+ * @prop {string} defaultShape default shape for symbol
+ * @prop {string} defaultShapeStrokeColor default symbol stroke color
+ * @prop {string} defaultShapeFillColor default symbol fill color
+ * @prop {string} defaultShapeSize default symbol shape size in px
+ * @prop {string} defaultPointType default point type
```

Comment:
please add values "symbol" or "marker"
```suggestion
 * @prop {string} defaultPointType default point type, can be "symbol" or "marker", default is "marker"
```